tools/internal_ci/linux/grpc_xds_bazel_test_in_docker.sh

```diff
@@ -47,9 +47,9 @@ touch "$TOOLS_DIR"/src/proto/grpc/testing/__init__.py
 
 bazel build test/cpp/interop:xds_interop_client
 
-"$PYTHON" tools/run_tests/run_xds_tests.py \
+GRPC_VERBOSITY=debug GRPC_TRACE=xds,xds_client "$PYTHON" tools/run_tests/run_xds_tests.py \
```

Comment:
You should actually set this to "xds_client,xds_resolver,cds_lb,xds_lb".

Reply:
Done, thanks